documents/policies/internal/Working Group Formation.md

```diff
@@ -0,0 +1,42 @@
+# WCA Working Group Formation Policy
+
+### Version 1.0 {.version}
+
+## Purpose
+The purpose of this policy is to provide guidelines for the formation and operation of working groups within the World Cube Association (WCA) in order to facilitate collaboration, innovation, and progress in specific focus areas.
+
+## Policy
+1. Definition of a Working Group
+   1. A working group is a temporary team of WCA Staff and/or community members established to address a specific project or area of interest within the organization. Working groups are tasked with generating ideas, providing recommendations, and executing initiatives related to their assigned focus area
+2. Formation
+   1. Working groups may be proposed by any WCA Staff member.
+   2. The Working group must be sponsored by a team leader, a Senior Delegate or a Board member.
```

Comment:
Edge case: what if a Sponsor loses that role between proposal and agreement? or while the working group is underway. Presumably a new sponsor would need to be nominated? E.g. hypothetically the sponsor was a Senior Delegate but their term expired in the intervening 2 months it took for the Board to review their email!?

Reply:
Also, I would recommend replacing "Board Member" with "Officer". We need to start laying the groundwork for non-executive directors to exist.

Reply:
Thanks changed Board member to Officer.
I am not worried about that edge case and I don't believe it needs to be defined in the policy

Reply:
also added Director

Reply:
I disagree, I don't think a Director alone should be able to do it. Directors should be focussed on governance not on being the people to get things done.